Makefile

```diff
@@ -0,0 +1,11 @@
+TEMURIN_TAGS := 8u322-b06-jdk 11.0.14.1_1-jdk, 8u322-b06-jre, 11.0.14.1_1-jre
+JOBS := $(addprefix temurin-,${TEMURIN_TAGS})
+
+all: ${JOBS}
+.PHONY: all ${JOBS}
+
+${JOBS}: temurin-%: Dockerfile
+	docker build \
+	  --build-arg TEMURIN_TAG=$* \
+	  --tag dwolla/docker-java:$*-SNAPSHOT \
+	  .
```

Comment:
This doesn't work for me locally, I think because there are the two Dockerfiles in different directories:

```
$ make 
make: *** No rule to make target `Dockerfile', needed by `temurin-8u322-b06-jdk'.  Stop.
```

Maybe we could separate the JDK and JRE tags into separate target groups? Something like this:

```make
TEMURIN_JDK_TAGS := 8u322-b06-jdk 11.0.14.1_1-jdk
TEMURIN_JRE_TAGS := 8u322-b06-jre 11.0.14.1_1-jre
JDK_JOBS := $(addprefix temurin-jdk-,${TEMURIN_JDK_TAGS})
JRE_JOBS := $(addprefix temurin-jre-,${TEMURIN_JRE_TAGS})

all: ${JDK_JOBS} ${JRE_JOBS}
.PHONY: all ${JDK_JOBS} ${JRE_JOBS}

${JDK_JOBS}: temurin-jdk-%: jdk/Dockerfile
	docker build \
	  --build-arg TEMURIN_TAG=$* \
	  --tag dwolla/docker-java:$*-SNAPSHOT \
	  jdk/

${JRE_JOBS}: temurin-jre-%: jre/Dockerfile
	docker build \
	  --build-arg TEMURIN_TAG=$* \
	  --tag dwolla/docker-java:$*-SNAPSHOT \
	  jre/
```

Reply:
that works!

Reply:
It would be nice to add `check` and `clean` targets as well, so we can check to make sure the tags in the Makefile match the tags in the GitHub Actions matrix, and we can easily clean up any snapshot images built locally. Something like this:

```make
TEMURIN_JDK_TAGS := 8u322-b06-jdk 11.0.14.1_1-jdk
TEMURIN_JRE_TAGS := 8u322-b06-jre 11.0.14.1_1-jre
JDK_JOBS := $(addprefix temurin-jdk-,${TEMURIN_JDK_TAGS})
JRE_JOBS := $(addprefix temurin-jre-,${TEMURIN_JRE_TAGS})
JDK_CHECKS := $(addprefix temurin-jdk-check-,${TEMURIN_JDK_TAGS})
JRE_CHECKS := $(addprefix temurin-jre-check-,${TEMURIN_JRE_TAGS})
JDK_CLEAN := $(addprefix temurin-jdk-clean-,${TEMURIN_JDK_TAGS})
JRE_CLEAN := $(addprefix temurin-jre-clean-,${TEMURIN_JRE_TAGS})

all: ${JDK_JOBS} ${JRE_JOBS}
check: ${JDK_CHECKS} ${JRE_CHECKS}
clean: ${JDK_CLEAN} ${JRE_CLEAN}
.PHONY: all check clean ${JDK_JOBS} ${JRE_JOBS} ${JDK_CHECKS} ${JRE_CHECKS} ${JDK_CLEAN} ${JRE_CLEAN}

${JDK_JOBS}: temurin-jdk-%: jdk/Dockerfile
	docker build \
	  --build-arg TEMURIN_TAG=$* \
	  --tag dwolla/docker-java:$*-SNAPSHOT \
	  jdk/

${JRE_JOBS}: temurin-jre-%: jre/Dockerfile
	docker build \
	  --build-arg TEMURIN_TAG=$* \
	  --tag dwolla/docker-java:$*-SNAPSHOT \
	  jre/

${JDK_CHECKS}: temurin-jdk-check-%:
	grep --silent "^          - $*$$" .github/workflows/ci.yml

${JRE_CHECKS}: temurin-jre-check-%:
	grep --silent "^          - $*$$" .github/workflows/ci.yml

${JDK_CLEAN}: temurin-jdk-clean-%:
	docker rmi dwolla/docker-java:$*-SNAPSHOT

${JRE_CLEAN}: temurin-jre-clean-%:
	docker rmi dwolla/docker-java:$*-SNAPSHOT
```